trompace/mutations/musicgroup.py

```diff
@@ -145,3 +144,14 @@ def mutation_musicgroup_remove_exact_match_musicgroup(identifier_from: str, iden
     Returns: a mutation to remove the exactMatch relationship from the MusicGroup objects
     """
     return format_link_mutation("RemoveMusicGroupExactMatch", identifier_from, identifier_to)
+
+
+def mutation_add_musicgroup_member(identifier_from: str, identifier_to: str):
```

Comment:
when we have a method that joins items of two different types I prefer to give them more descriptive names than from and to. The above methods (e.g. mutation_musicgroup_remove_exact_match_musicgroup) work with two MusicGroup objects, which is why it has generic names.

For this one, use `identifier_person` and `identifier_group`